index.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+<title>PDP</title>
+</head>
+<body>
+<div class="header">
+	<div class="top-header">
+		<div class="wrapper">
+			<div class="currency-selector-wrapper">
+				<select class="currency-selector" name="currency-selector">
+					<option value="usd">usd</option>
+					<option value="uah">uah</option>
+					<option value="rur">rur</option>
+				</select>
+			</div>
+			<div class="logo">
+				<a href="#">
+					<img src="images/logo.png" alt="logo"> 
+				</a>
+			</div>
+			<div class="cart-wrapper">
+				<a href="#">Cart (<span class="cart-quantity">3</span>)</a>
+			</div>
+		</div>
+	</div>
+	<div class="bottom-header">
+		<div class="wrapper">
+			<ul class="navigation">
+				<li><a href="#">Home</a></li>
+				<li><a href="#">Men</a>
+					<ul class="sub-navigation">
+						<li><a href="#">Jackets</a></li>
+						<li><a href="#">Shoes</a></li>
+						<li><a href="#">Coats</a></li>
+						<li><a href="#">Hats</a></li>
+					</ul>
+				</li>
+				<li><a href="#">woMen</a>
+					<ul class="sub-navigation">
+						<li><a href="#">Jackets</a></li>
+						<li><a href="#">Shoes</a></li>
+						<li><a href="#">Coats</a></li>
+						<li><a href="#">Hats</a></li>
+					</ul>
+				</li>
+				<li><a href="#">Lockbook</a></li>
+				<li><a href="#">about</a></li>
+				<li><a href="#">blog</a></li>
+			</ul>
+			<ul class="utility-menu">
+				<li><a href="#">wishlist</a></li>
+				<li><a href="#">search</a></li>
+			</ul>
+		</div>
+	</div>
+	<div class="advertisement-block">
+		<div class="wrapper">
+			<p>start selling your products or buy them from anywhere!</p>
+			<a href="#" class="close-button">close</a>
+		</div>
+	</div>
+</div>
+<div class="content">
+	<div class="wrapper">
+		<div class="product-image-container">
+			<div class="main-image-wrapper">
+				<img src="images/main-product-image.jpg" alt="main product image">
+			</div>
+			<ul class="thumbnails">
+				<li>
+					<a href="#" class="selected"><img src="images/product-thumbnail-1.jpg" alt="product thumbnail 1"></a>
+				</li>
+				<li>
+					<a href="#"><img src="images/product-thumbnail-2.jpg" alt="product thumbnail 2"></a>
+				</li>
+				<li>
+					<a href="#"><img src="images/product-thumbnail-3.jpg" alt="product thumbnail 3"></a>
+				</li>
+				<li>
+					<a href="#"><img src="images/product-thumbnail-4.jpg" alt="product thumbnail 4"></a>
+				</li>
+			</ul>
+		</div>
+		<div class="product-details-container">
+			<p class="product-text">Back to <a href="#" class="for-text-links">Woman<a/></p>
```

Comment:
1. it is preferably to use div instead of p here, because it is a functional block, not text.
2. classname "for-text-links" isn't good - it is not clear what is the purpose of this link.